=== modified file 'gtk-3.0/gtk/gtk.h'
--- gtk-3.0/gtk/gtk.h	revid:mats@linuxfoundation.org-20140621143624-skc4nuh9uzjhbm55
+++ gtk-3.0/gtk/gtk.h	2014-06-21 14:36:59 +0000
@@ -793,6 +793,8 @@
 
     typedef struct _GtkPageSetup GtkPageSetup;
 
+    typedef struct _GtkWidget GtkWidget;	   /* XXX move from below */
+
     typedef gint(*GtkKeySnoopFunc) (GtkWidget * grab_widget,
 				    GdkEventKey * event,
 				    gpointer func_data);
@@ -825,7 +827,7 @@
 
     typedef struct _GtkTooltip GtkTooltip;
 
-    typedef struct _GtkWidget GtkWidget;
+    /* typedef struct _GtkWidget GtkWidget;		XXX move up */
 
     typedef struct _GtkWidgetPath GtkWidgetPath;
 
@@ -847,6 +849,16 @@
 
     typedef struct _GtkTreeSelectionPrivate GtkTreeSelectionPrivate;
 
+    typedef struct _GtkCellRenderer GtkCellRenderer;  /* XXX move from below */
+
+    typedef struct _GtkTreeIter GtkTreeIter;	      /* XXX move from below */
+
+    typedef struct _GtkTreePath GtkTreePath;	      /* XXX move from below */
+
+    typedef struct _GtkTreeModel GtkTreeModel;	      /* XXX move from below */
+
+    typedef struct _GtkTreeSelection GtkTreeSelection;/* XXX move from below */
+
     typedef gboolean(*GtkTreeSelectionFunc) (GtkTreeSelection * selection,
 					     GtkTreeModel * model,
 					     GtkTreePath * path,
@@ -1334,7 +1346,7 @@
 	GTK_CELL_RENDERER_MODE_EDITABLE
     } GtkCellRendererMode;
 
-    typedef struct _GtkCellRenderer GtkCellRenderer;
+    /* typedef struct _GtkCellRenderer GtkCellRenderer;  XXX move up */
 
     typedef struct _GtkCellRendererPrivate GtkCellRendererPrivate;
 
@@ -2338,7 +2350,7 @@
 
     typedef struct _GtkTreeViewPrivate GtkTreeViewPrivate;
 
-    typedef struct _GtkTreeSelection GtkTreeSelection;
+    /* typedef struct _GtkTreeSelection GtkTreeSelection;   XXX move up */
 
     typedef struct _GtkTreeSelectionClass GtkTreeSelectionClass;
 
@@ -2406,13 +2418,13 @@
 	GTK_LICENSE_ARTISTIC
     } GtkLicense;
 
-    typedef struct _GtkTreeIter GtkTreeIter;
+    /* typedef struct _GtkTreeIter GtkTreeIter;		XXX move up */
 
-    typedef struct _GtkTreePath GtkTreePath;
+    /* typedef struct _GtkTreePath GtkTreePath;		XXX move up */
 
     typedef struct _GtkTreeRowReference GtkTreeRowReference;
 
-    typedef struct _GtkTreeModel GtkTreeModel;
+    /* typedef struct _GtkTreeModel GtkTreeModel;	XXX move up */
 
     typedef struct _GtkTreeModelIface GtkTreeModelIface;
 

