=== modified file 'gtk-3.0/gtk/gtk.h'
--- gtk-3.0/gtk/gtk.h	revid:mats@linuxfoundation.org-20140619214605-xixla3qtz48b4enp
+++ gtk-3.0/gtk/gtk.h	2014-06-21 12:49:02 +0000
@@ -792,6 +792,8 @@
 
     typedef struct _GtkPageSetup GtkPageSetup;
 
+    typedef struct _GtkWidget GtkWidget;	   /* XXX move from below */
+
     typedef gint(*GtkKeySnoopFunc) (GtkWidget * grab_widget,
 				    GdkEventKey * event,
 				    gpointer func_data);
@@ -824,7 +826,7 @@
 
     typedef struct _GtkTooltip GtkTooltip;
 
-    typedef struct _GtkWidget GtkWidget;
+    /* typedef struct _GtkWidget GtkWidget;		XXX move up */
 
     typedef struct _GtkWidgetPath GtkWidgetPath;
 
@@ -846,6 +848,16 @@
 
     typedef struct _GtkTreeSelectionPrivate GtkTreeSelectionPrivate;
 
+    typedef struct _GtkCellRenderer GtkCellRenderer;  /* XXX move from below */
+
+    typedef struct _GtkTreeIter GtkTreeIter;	      /* XXX move from below */
+
+    typedef struct _GtkTreePath GtkTreePath;	      /* XXX move from below */
+
+    typedef struct _GtkTreeModel GtkTreeModel;	      /* XXX move from below */
+
+    typedef struct _GtkTreeSelection GtkTreeSelection;/* XXX move from below */
+
     typedef gboolean(*GtkTreeSelectionFunc) (GtkTreeSelection * selection,
 					     GtkTreeModel * model,
 					     GtkTreePath * path,
@@ -1333,7 +1345,7 @@
 	GTK_CELL_RENDERER_MODE_EDITABLE
     } GtkCellRendererMode;
 
-    typedef struct _GtkCellRenderer GtkCellRenderer;
+    /* typedef struct _GtkCellRenderer GtkCellRenderer;  XXX move up */
 
     typedef struct _GtkCellRendererPrivate GtkCellRendererPrivate;
 
@@ -2330,7 +2342,7 @@
 
     typedef struct _GtkTreeViewPrivate GtkTreeViewPrivate;
 
-    typedef struct _GtkTreeSelection GtkTreeSelection;
+    /* typedef struct _GtkTreeSelection GtkTreeSelection;   XXX move up */
 
     typedef struct _GtkTreeSelectionClass GtkTreeSelectionClass;
 
@@ -2398,13 +2410,13 @@
 	GTK_LICENSE_ARTISTIC
     } GtkLicense;
 
-    typedef struct _GtkTreeIter GtkTreeIter;
+    /* typedef struct _GtkTreeIter GtkTreeIter;		XXX move up */
 
-    typedef struct _GtkTreePath GtkTreePath;
+    /* typedef struct _GtkTreePath GtkTreePath;		XXX move up */
 
     typedef struct _GtkTreeRowReference GtkTreeRowReference;
 
-    typedef struct _GtkTreeModel GtkTreeModel;
+    /* typedef struct _GtkTreeModel GtkTreeModel;	XXX move up */
 
     typedef struct _GtkTreeModelIface GtkTreeModelIface;
 

