=== modified file 'gtk-3.0/gdk/gdk.h'
--- gtk-3.0/gdk/gdk.h 	2014-06-21 08:45:21.771432847 -0600
+++ gtk-3.0/gdk/gdk.h	2014-06-21 08:46:17.947251520 -0600
@@ -277,6 +277,8 @@
 
     typedef struct _GdkGeometry GdkGeometry;
 
+    typedef struct _GdkWindow GdkWindow;   /*XXX hand-edit: move from below */
+
     typedef struct _GdkWindowAttr GdkWindowAttr;
 
     typedef struct _GdkWindowRedirect GdkWindowRedirect;
@@ -527,7 +529,7 @@
 
     typedef struct _GdkScreen GdkScreen;
 
-    typedef struct _GdkWindow GdkWindow;
+    /* typedef struct _GdkWindow GdkWindow;	XXX hand-edit: move up */
 
     typedef struct _GdkKeymap GdkKeymap;
 
